### PR TITLE
update i18Next

### PR DESCRIPTION
**Pull request by zacheryconverse**
Bump i18Next from v19 to v21. Allows the use of JSON format v4 plural handling

### CODE REVIEW

**Comment by zacheryconverse** [comment]
@petyosi The way we handle replies is now updated.

File: src/components/Message/MessageRepliesCountButton.tsx [hunk 1]
```diff
@@ -25,17 +25,15 @@ const UnMemoizedMessageRepliesCountButton: React.FC<MessageRepliesCountButtonPro
...
   }
 
   if (reply_count && reply_count > 1) {
     if (labelPlural) {
       pluralReplyText = `${reply_count} ${labelPlural}`;
     } else {
-      pluralReplyText = t('{{ replyCount }} replies', {
-        replyCount: reply_count,
-      });
+      pluralReplyText = t('replyCount_other', { replyCount: reply_count });
```
Comment (petyosi) [inline src/components/Message/MessageRepliesCountButton.tsx:36]: Based on what [I am seeing here](https://www.i18next.com/translation-function/plurals#singular-plural), you're not supposed to use `key_one` or `key_other` as a key. Instead, you should use just `key` and let the framework pull the right key based on the json provided and the count available.

**Review by petyosi — CHANGES_REQUESTED** [review]
If I understand [the samples from the code](https://www.i18next.com/translation-function/plurals#singular-plural), We should not use `key_one` and `key_other` in the source code.

File: examples/capacitor/yarn.lock [hunk 2]
```diff
@@ -9799,6 +9814,14 @@ react-virtuoso@^2.2.1, react-virtuoso@^2.8.2:
     "@virtuoso.dev/react-urx" "^0.2.12"
     "@virtuoso.dev/urx" "^0.2.12"
 
+react-virtuoso@^2.3.1:
```
Comment (petyosi) [inline examples/capacitor/yarn.lock:9817]: Can we remove the example yarn bumps? Not like any harm, but it pollutes the PR?

File: src/components/Message/MessageRepliesCountButton.tsx [hunk 3]
```diff
@@ -18,41 +18,26 @@ const UnMemoizedMessageRepliesCountButton: React.FC<MessageRepliesCountButtonPro
...
-    return (
-      <button
-        className='str-chat__message-replies-count-button'
-        data-testid='replies-count-button'
-        onClick={onClick}
-      >
-        <ReplyIcon />
-        {reply_count === 1 ? singleReplyText : pluralReplyText}
-      </button>
-    );
```
Comment (petyosi) [inline src/components/Message/MessageRepliesCountButton.tsx:52]: Lovely change.

**Review by petyosi — APPROVED** [review]

**Review by arnautov-anton — APPROVED** [review]

File: examples/capacitor/yarn.lock (hunk 2 above)
Comment (MartinCupela) [reply examples/capacitor/yarn.lock]: Removed via git checkout

**Pull request merged**